.github/workflows/ci.yaml

```diff
@@ -6,9 +6,9 @@ on:
 name: ci
 env:
   GOOGLE_CLOUD_PROJECT: "span-cloud-testing"
-  GOOGLE_CLOUD_INSTANCE: "spanner-testing"
+  GOOGLE_CLOUD_INSTANCE: "pgadapter-testing"
   GOOGLE_CLOUD_DATABASE: "testdb_e2e_psql"
-  GOOGLE_CLOUD_ENDPOINT: "staging-wrenchworks.sandbox.googleapis.com"
+  GOOGLE_CLOUD_ENDPOINT: "spanner.googleapis.com"
```

Comment:
We should not use the staging endpoint anymore, for two reasons:
- Currently, the staging endpoint returns an `INVALID_ARGUMENT` for multiple admin calls (this is temporary, and so not a permanent reason not to use it)
- The staging environment can contain features that will not be available on the production endpoint, and we should ensure that PgAdapter only uses features that are actually available on production.